sdk/src/main/java/co/decodable/sdk/pipeline/DecodableSecret.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright Decodable, Inc.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package co.decodable.sdk.pipeline;
+
+import co.decodable.sdk.pipeline.internal.DecodableSecretImpl;
+import java.util.Date;
+
+public interface DecodableSecret {
```

Comment:
Being part of the SDK's public API, this should have JavaDoc on the class-level as well as all public members.

Reply:
Added JavaDocs